docs/topics/db/optimization.txt

```diff
@@ -250,8 +250,8 @@ Use ``QuerySet.count()``
 Use ``QuerySet.exists()``
 -------------------------
 
-...if you only want to find out if at least one result exists, rather than ``if
-queryset``.
+...if you only want to find out if at least one result exists, rather than 
+``if queryset``.
```

Comment:
Small unrelated change: Reformatted this line slightly so that it didn't break syntax-highlighting in my editor.

Reply:
Done.